papers/donath/donath-pylira.rst

```diff
@@ -0,0 +1,502 @@
+
+.. math::
+   :label: rl
+
+   \mathbf{x}_{n + 1}  = \mathbf{x}_{n} -\alpha \cdot \frac{\partial \mathcal{C}\left( \mathbf{d} | \mathbf{x} \right)}{\partial x_i}
+
+Where :math:`\alpha` is a factor to define the step size. This method is in general
+equivalent to the gradient decent and backpropagation methods used in modern machine
+learning techniques. This basic principle of solving the deconvolution problem for
+images with Poisson noise was proposed by :cite:`Richardson1972` and :cite:`Lucy1974`.
+Their method, named after the original authors, is often known as the *Richardson & Lucy* (RL)
+method. It was shown by :cite:`Richardson1972` that this converges to a maximum
+likelihood solution of Eq. :ref:`cash`. A Python implementation of the standard RL method
+is available e.g. in the `Scikit-Image` package :cite:`skimage`.
+
+Instead of the iterative, gradient decent based optimization it is also possible to sample from
+the likelihood function using a simple Metropolis-Hastings approach. This is demonstrated
+in one of the *Pylira* online tutorials (`Introduction to Deconvolution using MCMC Methods <https://pylira.readthedocs.io/en/latest/pylira/user/tutorials/notebooks/mcmc-deconvolution-intro.html>`__).
+
+
+RL Reconstruction Quality
++++++++++++++++++++++++++
+While technically the RL method converges to a maximum likelihood solution, it mostly
+still results in poorly restored images, especially if extended emission regions are
+present in the image. The problem is illustrated in Fig. :ref:`rl` using
+a simulated example image. While for a low number of iterations the RL method
+still results in a smooth intensity distribution, the structure of the image
+decomposes more and more into a set of point-like sources with growing number
+of iterations.
+
+Because of the PSF convolution an extended emission region
+can decompose into multiple nearby point sources and still lead to good model prediction,
+when compared with the data. Those almost equally good solutions correspond
+to many narrow local minima or "spikes" in the global likelihood surface. Depending
+on the start estimate for the reconstructed image :math:`\mathbf{x}` the RL method will follow
+the steepest gradient and converge towards the nearest narrow local minimum.
+This problem has been described by multiple authors, such as :cite:`Reeves1994`
+and :cite:`Fish95`.
+
+
+.. figure:: images/richardson-lucy.png
+   :scale: 90%
+   :figclass: bht
+
+   The images show the result of the RL algorithm applied to a simulated example dataset
+   with varying numbers of iterations. The image in the upper left shows the simulated
+   counts. Those have been derived from the ground truth (upper mid) by convolving with
+   a Gaussian PSF of width :math:`\sigma=3~\mathrm{pix}` and applying Poisson noise to
+   it. The illustration uses the implementation of the RL algorithm from the `Scikit-Image`
+   package :cite:`skimage`.  :label:`rl`
+
+
+LIRA Multi-Scale Prior
```

Comment:
The title appears to be too narrow.  LIRA is a multiscale Bayesian reconstruction algorithm that *uses* multi-scale priors.  This gives the impression that the priors are the main advance made by LIRA